src/footprint/azext_footprint/generated/_help.py

```diff
@@ -19,7 +19,7 @@
 
 helps['footprint profile list'] = """
     type: command
-    short-summary: Retrieves the information about all Footprint profiles under a subscription.
+    short-summary: "Retrieves the information about all Footprint profiles under a subscription."
```

Comment:
Can I know why we need to add double quotes here?

Reply:
I didn't do this manually, regenerating added these to the file.

Reply:
I add quote mark for each description because a previous issue that if the description contains : the cli will fail to load the help info.

Reply:
@qiaozha Hi. Could you share more context about this change?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/7006fdd5383c11df9eb5fa09ea05100c5578e854/specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPacks_API.json#L225 Like this one.

Reply:
@MyronFanQiu I'd like to merge ASAP to provide the new version to our internal customers, please ping me (at any hour) if anything else needs to be addressed